howtos/setting_up_codacy_with_github.md

```diff
@@ -0,0 +1,94 @@
+# How to setup Codacy for an Eiffel project
+
+This walkthrough is based on Emelie Pettersons walkthrough on [Eiffel Intelligence issue #320](https://github.com/eiffel-community/eiffel-intelligence/pull/320)
+
+You login with you GitHub account. When you login Codacy will request for some permissions to your
+ GitHub account. This is to able to fetch all you public repositories into Codacy. For more
+ information take a look at their help page [Which permissions does Codacy need from my account?](https://support.codacy.com/hc/en-us/articles/115003405529-Which-permissions-does-Codacy-need-from-my-account-)
+
+After you have logged in you will need to import project to work with. If this is the first time
+ it is recommended to start by importing your own fork to try out. You will however need to import
+ the real project later on.
+
+When you have imported a project, Codacy will put an notification on each pull request in the
+ status part. If you want to have more interaction checkout the *Integrations* tab under the
+ Setting page. In GitHub you can find this integration in *Webhooks* tab under the Settings page.
+
+At this point you have integrated Codacy into GitHub so it is performing style checks for your
+ project on each Pull Request as this it the default behavior for the Webhook.
+
+For here on it is assumed we are working the project and not the fork
+
+## Transfer the project
+
+To enable other people to change settings you will have to transfer the project to an organization.
+ The recommendation is to name the organization same as the maintainers team in GitHub. If there
+ is no organization to fit you need you will need to [create one](https://support.codacy.com/hc/en-us/articles/360009209594-Creating-and-renaming-an-organization).
+ When you have created the organization be sure to invite members from the GitHub maintainers
+ team. The [transfer](https://support.codacy.com/hc/en-us/articles/207994835-How-do-I-transfer-delete-a-project-from-my-account-)
+ option is found in the setting page for the project.
+
+After you have transferred the project and want to make changes, you will first have to navigate to
+ the organizations page and then to the project to access the settings page.
+
+## Add coverage
+In order to start showing coverage report for the project you will probably first time need to
```

Comment:
Reverse the order of expressions here.
'need to upload the coverage manually the first time'

Reply:
Done